sites/docs/pages/components/date-range.md

```diff
@@ -138,6 +159,19 @@ A manually specified end date to use for the range
 Title to display in the Date Range component
 
 </PropListing>
+
+
+<PropListing 
+    name="presetRanges"
+    options={['last7Days', 'last30Days', 'last90Days', 'last3Months', 'last6Months', 'last12Months', 'lastMonth', 'lastYear', 'monthToDate', 'yearToDate', 'allTime']}
```

Comment:
these will not copy as expected as they will not include the curly braces or square braces